privatelink/aws/debug-connectivity.sh

```diff
@@ -105,7 +105,7 @@ fmt="%-5s %s\n"
 
 # shellcheck disable=SC2001
 httpsname="https://$(echo "$bootstrap" | sed -e 's/:.*//')/kafka/v3/clusters/lkc-test"
-httpsout=$(curl --silent --include "$httpsname")
```

Comment:
5 seconds seem to be short. how about 15?

Reply:
Since we're running this in a datacenter, 15 seconds seems too long to me? Just TCP connect should take single milliseconds. Is there something else that could cause such a long delay?

Reply:
Not particularly. I expect  `--connect-timeout` to include name resolution?

Reply:
You are right @fzhong-connect .

> --connect-timeout limits the time curl spends trying to connect to the host. All the necessary steps done before the connection [...] have to be completed within the given time frame. Failing to connect within the given time causes curl to exit with a timeout exit code (28). The steps done before a connect is considered successful include DNS lookup and subsequent TCP, TLS or QUIC handshakes.

Though I doubt all of this would take longer than a few hundred ms. I am open to either 5s or 15s.

Reply:
Lets go with 15, its already better

Reply:
Done